scripts/generate-comp-ver-table.py

```diff
@@ -0,0 +1,132 @@
+    with open(file_path, "r", encoding="utf-8") as stream:
+        try:
+            values = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+            sys.exit(1)
+
+        components_versions = {
+            'OpenJDK': values.get('dependencies_version', {}).get('jdk'),
+            'Apache_Tomcat': values.get('dependencies_version', {}).get('tomcat'),
+            'PostgreSQL': values.get('dependencies_version', {}).get('postgres_major_version'),
+            'Apache_ActiveMQ': values.get('dependencies_version', {}).get('activemq'),
+            'Repository': values.get('acs', {}).get('version'),
+            'Share': values.get('acs', {}).get('version'),
+            'Search_Services': values.get('search', {}).get('version'),
+            'All-In-One_Transformation_Engine': values.get('transform', {}).get('version'),
+            'AOS': values.get('amps', {}).get('aos_module', {}).get('version'),
+            'GoogleDocs': values.get('amps', {}).get('googledrive_repo', {}).get('version'),
+            'Digital_Workspace': values.get('adw', {}).get('version'),
+            'Transform_Router': values.get('trouter', {}).get('version'),
+            'Shared_File_Store': values.get('sfs', {}).get('version'),
+            'Sync_Service': values.get('sync', {}).get('version'),
+        }
+        return components_versions
+
+
+def get_content_of_new_table():
+    """Generate content of new table"""
+    end_table = ""
+    first_line = get_first_line_of_table()
+    end_table+=f"{first_line}\n"
+    second_line = '|-|-|-|-|-|'
+    end_table+=f"{second_line}\n"
+
+    for component in list(get_components_versions("group_vars/all.yml").keys()):
+        line_in_table = ''
+        line_in_table+=f"| {component.replace('_', ' ')} | "
+
+        for values_file in get_acs_extra_values_files():
+            components_versions = get_components_versions(values_file)
+            version = (components_versions[component])
+
+            if values_file == 'community-extra-vars.yml' and component in ['Digital_Workspace', 'Transform_Router', 'Shared_File_Store', 'Sync_Service']:
+                version = 'N/A'
+            elif values_file == 'community-extra-vars.yml' and component in ['AOS', 'GoogleDocs']:
+                version = ''
+            #use values from group_vars if None
+            elif version is None:
+                components_versions = get_components_versions(get_acs_extra_values_files()[0])
+                version = (components_versions[component])
+
+            if component == 'PostgreSQL':
+                version= str(version) + '.x'
+
+            line_in_table+=f"{version} | "
+
+        line_in_table=line_in_table[:-1]
+        end_table+=f"{line_in_table}\n"
+    return end_table
+
+def modify_table():
+    """Delete old table from README.md, and put new one"""
+    delete_next_lines = False
+    a_file = open("docs/README.md", "r", encoding="utf-8")
```

Comment:
Does this pass pylint? I thought with syntax was now the way to go with python3

Reply:
Passed pylint, i can change to "with open"